src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/XmlDataListSource.cs

```diff
@@ -154,5 +174,35 @@ public XmlNotesConfigurationField()
 </div>", true)
             { }
         }
+
+
+        /// <summary>
+        /// Returns an instance of XmlNamespaceManager with a specified XmlDocument.NameTable.  
+        /// Returns null if no namespace is defined.
+        /// </summary>
+        /// <param name="doc">The XmlDocument</param>
+        /// <returns>XmlNamespaceManager if there is at least one namespace, null if there is no namespace defined.</returns>
+        private XmlNamespaceManager CreateNamespaceManager(XmlDocument doc)
+        {
+            var result = new XmlNamespaceManager(doc.NameTable);
+
+            IDictionary<string, string> localNamespaces = null;
+            XPathNavigator xNav = doc.CreateNavigator();
+            while (xNav.MoveToFollowing(XPathNodeType.Element))
+            {
+                localNamespaces = xNav.GetNamespacesInScope(XmlNamespaceScope.Local);
+                foreach (var localNamespace in localNamespaces)
+                {
+                    string prefix = localNamespace.Key;
+                    if (string.IsNullOrEmpty(prefix))
+                        prefix = "DEFAULT";
```

Comment:
I'm curious, why `"DEFAULT"`? (and does this have any implications on the XPath queries?)

From looking at the MS docs, the default namespace prefix is an empty string.
<https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlnamespacemanager.addnamespace?view=netframework-4.7.2#remarks>